### ISSUE TITLE
Discussion forum / communication platform for crowdsourcing community

### ISSUE THREAD

**Issue by elainekamlley** [LibraryOfCongress/concordia#50]
Acceptance Criteria: 

- As a volunteer, I want to discuss something I've transcribed or reviewed or I want to ask a general question about Make Your Make so that I can better understand what I'm working on and communicate with other volunteers and the Community Managers. 

Expected behavior: click a "Discuss" button in the navigation bar to go to History Hub. Ideally a link to the page I was transcribing or the Campaign I am interested in will be populated into the History Hub discussion.

**Comment by VVH** [comment]
Things we still need to figure out 

- how we get volunteers from our site to HH. Do we go to just one landing page with instructions on how they should contact specific people/teams i.e. curator v. Community Managers
- or do we try to create a process at our end that grabs the link from the page they're on or the asset they are transcribing, and then use that to populate a link over on HH:

**Comment by colefemeno** [comment]
@vvh would like to see integrating the latter into the Contact form - possibly as an option "I'd like to discuss this page" which could result in the form giving way to a "join us on HH" invitation. This approach might require reorganizing the fields (as a decision tree) to categorize a question or need before providing contact information.

**Comment by VVH** [comment]
@colefemeno can you give us an update here about where we are with HH? I'm envisioning a distinction between contact us (email community managers) and discuss (a forum where people can chat in the open)

Optimal: 
1) User wants to discuss a page they've just transcribed/reviewed/viewed. They click a "Discuss" button from within the transcription/review/view interface. They go directly to HH in a new tab. The link for the page they've just transcribed goes with them and populates their new comment. 

2) User wants to email a community manager.  They click "Contact Us" and a form appears on screen populated with their email and the link for the page they're on. 

MVP for full launch: 
1) A "Discuss" button opens link to HH in new tab. User has to copy and paste the link from the page they want to discuss. 

2) The form pops up and the volunteer has to populate with their email and the link for the relevant page. I think @acdha and @elainekamlley have discussed a solution that gets us closer to Optimal 2, but not sure where we are on Optimal 1.

**Comment by colefemeno** [comment]
Correct @VVH!  These appear to be separate user flows. Let's write these as new issues and close this one. Suggestions below!

- As a user, I want to discuss something that I've uncovered so that others can learn, too. 
- As a user, I want to give an example and ask how other people have done this work, to better understand instructions from the community mangers so that I can transcribe correctly 
- As a user, I want to contact a community manager with a pressing question via the Contact Us form so that I can get a swift reply. 

Expected behavior:
The first user story can be performed within HH. 
The second user story moves the user from the Contact Us form to perform the task in HH. 
The third user story can be performed in the Contact Us form. 

Desired:
- I should be able to access HH from the transcription page, About/FAQ page, and Contact Us form.
- I should be able to categorize my question in the Contact Us form to best route the question. 
- I should be able to tell what kind of response I will get based on the way I am seeking to discuss what I have encountered.
- Contact Us form would autopopulate the links and my email
